Connection.php

```diff
@@ -287,12 +287,21 @@ protected function _run(HoaSocket\Node $node)
                             break;
                         }
 
-                        $this->getListener()->fire(
-                            'message',
-                            new Event\Bucket([
-                                'message' => $frame['message']
-                            ])
-                        );
+                        try {
```

Comment:
The upper try/catch close the current node connection.

It's especially what we have to fix, and the original goal of https://github.com/hoaproject/Socket/pull/32.